src/sparse/python_binding.cc

```diff
@@ -0,0 +1,25 @@
+/*!
+ *  Copyright (c) 2022 by Contributors
+ * \file sparse/pybind.cc
+ * \brief DGL sparse library Python binding
+ */
+#include <torch/custom_class.h>
+#include <torch/script.h>
+
+#include "./elementwise_op.h"
+#include "./sparse_matrix.h"
+
+namespace dgl {
+namespace sparse {
+
+TORCH_LIBRARY(dgl_sparse, m) {
+  m.class_<SparseMatrix>("SparseMatrix")
+      .def("coo", &SparseMatrix::COOTensors)
+      .def("val", &SparseMatrix::Value)
+      .def("shape", &SparseMatrix::Shape)
+      .def("__add__", &Add);
+  m.def("create_from_coo", &CreateFromCOO);
+}
```

Comment:
I will suggest to move the torchscript registrations at the end of the original files, for example in this case move it to `sparse_matrix.cc` and remove this file.

Reply:
I agree. A minor issue is we need to use different namespace names if we register them in different files.

Reply:
I think I will move them after we decide how to bind C++ SparseMatrix to Python.